### ISSUE TITLE
MeshSync Blender DCC Plugin not working on M1 Max Macbook

### ISSUE THREAD

**Issue by erik-sutton95** [Unity-Technologies/MeshSyncDCCPlugin#198]
Can not enable/install the Blender MeshSync plugin on my Macbook with the M1 Max chip.

<img width="1506" alt="image" src="https://user-images.githubusercontent.com/9084979/170282031-c3e96331-40d7-4272-8216-00e172660b49.png">

**Comment by sindharta** [comment]
Yes, Blender Arm64 (for M1) build is not supported yet, but you should be able to use Blender x64 build on your M1 machine, and MeshSync will work with this version.

**Comment by erik-sutton95** [comment]
Alright, thank you! Will use this until Arm64 is supported 👍
